README.md

```diff
@@ -1,130 +1,181 @@
 
-Fission helps developers build and scale their apps. We’re building a web native file system that combines files, encryption, and identity, like an open source iCloud.
+The Webnative SDK empowers developers to build fully distributed web applications without needing a complex back-end. The SDK provides:
 
----
+- **User accounts** via the browser's [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) or by using a blockchain wallet as a [webnative plugin](https://github.com/fission-codes/webnative-walletauth).
+- **Authorization** using [UCAN](https://ucan.xyz/).
+- **Encrypted file storage** via the [Webnative File System](https://guide.fission.codes/developers/webnative/file-system-wnfs) backed by [IPLD](https://ipld.io/).
+- **Key management** via websockets and a two-factor auth-like flow.
 
-**[Read the Guide for extended documentation and getting started information](https://guide.fission.codes/developers/webnative)**.  
-The API reference can be found at [webnative.fission.app](https://webnative.fission.app)
+Webnative applications work offline and store data encrypted for the user by leveraging the power of the web platform. You can read more about Webnative in Fission's [Webnative Guide](https://guide.fission.codes/developers/webnative). There's also an API reference which can be found at [webnative.fission.app](https://webnative.fission.app)
 
----
 
-## What you'll find here
 
-The Fission Webnative SDK offers tools for:
-- authenticating through a Fission **authentication lobby**  
-  (a lobby is where you can make a Fission account or link an account)
-- managing your web native **file system**  
-  (this is where a user's data lives)
-- tools for building DIDs and UCANs.
-- interacting with the users apps via the **platform APIs**
+# Getting started
 
 ```ts
-// ES6
-import * as wn from 'webnative'
+// ESM
+import * as wn from "webnative"
 
 // Browser/UMD build
 const wn = globalThis.webnative
 ```
 
-# Authentication
+## Creating a Program
+
+A Webnative program is an assembly that makes up a part of, or a whole distributed web application. It consists out of several components that can be customised. _We'll stick with the default components for now, which means we'll be using the Web Crypto API._
 
 ```ts
-const state = await wn.initialise({
-  permissions: {
-    // Will ask the user permission to store
-    // your apps data in `private/Apps/Nullsoft/Winamp`
-    app: {
-      name: "Winamp",
-      creator: "Nullsoft"
-    },
-
-    // Ask the user permission to additional filesystem paths
-    fs: {
-      private: [ wn.path.directory("Audio", "Music") ],
-      public: [ wn.path.directory("Audio", "Mixtapes") ]
-    }
+const program = await wn.program({
+  // Can also be a string, used as an identifier for caches.
+  // If you're developing multiple apps on the same localhost port,
+  // make sure these differ.
+  id: { creator: "Nullsoft", name: "Winamp" }
```

Comment:
Mentioned in the WAT PR the idea of using `namespace` in place of `tag` or `id`. what do you think?
```suggestion
  namespace: { creator: "Nullsoft", name: "Winamp" }
```

Reply:
Oh great idea! That's the word I was looking for 👌

Reply:
Updated to `namespace`